house-snark/src/relations/blender/types.rs

```diff
@@ -27,6 +27,8 @@ pub(super) type BackendMerklePath = Vec<CircuitField>;
 pub(super) type BackendLeafIndex = CircuitField;
 pub(super) type BackendAccount = CircuitField;
 
+pub type FrontendMerklePathSingle = [u64; 4];
```

Comment:
please keep frontend types together; and maybe rename to `FrontendMerklePathNode` or something like this

Reply:
Done